---
title: Patch notes of The Division 2
meta:
    description: All patch notes from the release on forward of The Division 2
---

This page contains all the patch notes that were published since the release of the game. Newest patch notes are always at the top.

## Patch from 11. April 2019

* Fixed not being able to fast travel to the Castle settlement.
* Fixed several cases of abnormal Bounty boss skill-use behaviour.
* Fixed an issue causing the Nemesis crafting materials to be unobtainable.
* Fixed Delta-03 error occurring at the end of Conflict matches.
* Fixed an issue that would cause the pathfinding line to behave abnormally when in close proximity to your destination.
* Fixed the "donating" sound playing repeatedly after having donated and abandoned a daily mission.
* Fixed an issue related to cache countdown timers.
* Fixed an issue where Specialization ammunition drop rate was lower than intended.

## Patch from 5. April 2019, Invasion: Battle for D.C.

* New content
** Invasion: Battle for D.C. is now live. You can find a content overview in [this article](https://tomclancy-thedivision.ubisoft.com/game/en-us/news-updates/346408/tidal-basin-is-coming).

* Weapons, Mods and Gear
** Overhauled the Weapon mod system, see the table at the end of the patch notes for detailed information. (Or our [weapon mods list](/weapon-mods.html))
** Fixed a wrong long-range damage value for M4 (super 90) and P416. Previously, they could sometimes increase damage over longer ranges instead of the other way around.
** Rifles:
*** Damage on the MK17 rifle reduced by 14%, also corrected this weapon not having a damage falloff over distance.
*** LW M4 damage increased by 5%, RPM increased from 240 to 360.
*** LVOA-C RPM increased from 240 to 380.
** Marksman Rifles
*** Model 700 damage reduced by 13%.
** Shotguns
*** AA12 damage increased by 16%.
** LMGs
*** MG5 damage increased by 12% and fixed reversed falloff, causing the weapon to do more damage over a longer range.
** Updated initial accuracy on the Sig Sniper MCmillan T1. It should now be much more reliable to shoot as soon as it’s shouldered and the transition to scoped view is over.
** Reduced the amount of Critical Damage and Headshot Damage that can roll on Gear, including the amount on existing Gear. The actual values will differ depending on the level and quality of the gear it occurs on. This is partially in preparation for another world tier, where we felt these values were already at the limit of what was healthy in WT4.
** Complete overhaul of skill mods to give more achievable requirements and more reasonable bonuses.
*** A closer equivalence between required skill power and the bonuses granted.
*** Changed the amount of skill power on gear to not increase exponentially with level, but instead more linearly. This results in most skill power values on gear being lower, but in accordance with the lowered requirements.
*** Tweaked the ranges of bonuses to be within acceptable bounds where they had extreme values before due to scaling errors. Most notably radius and ammo/charges values.
*** Players will see both the bonuses and the requirements of almost all skill mods change as a result of this overhaul.
*** In end game, both Superior and High-End skill mods exist to cover the different needs of different builds. They can overlap in power, but High-End mods have a higher maximum roll potential.
*** We are looking at ways to provide lower value/quality mods in end game world tiers in the future to provide a solution to the fact that on very low skill power builds, there would be room for those mods, and currently we are aware that means you have to save mods from your leveling experience.

* Player Skills
** Fixed an issue which caused the Bombardier Drone to get destroyed while climbing up/down a ladder or rope.
** Improved feedback of Bombardier Drone crashing into obstacles during the bombing phase.
** Fixed an issue with Firefly payload sometimes failing to activate when reaching its target.
** Fixed an issue where Revive hive would not self-deploy if the agent was busy performing a different action such as using different skill when it triggered.
** Revamped the Sniper Turret version of the turret skill.
*** It now will automatically target the hostile nearest to the players’ center-of-screen (the same method it uses to pick other turret targets) and will fire at that target when the button is pressed, without a lock-on order needed.
*** If a player aims at a hostile, the sniper turret will attempt to match their aim, allowing precision shots using the sniper turret.
*** An issue where the sniper turret wasn’t correctly getting bonus damage when hitting headshots has been fixed.
** Revamped the Chem launcher skill platform.
*** Pressing the skill button will now equip the chem launcher similar to swapping to a weapon. The player can aim and shoot with normal weapon controls, as well as run with the skill and blindfire it.
*** Quick deploy at the agents own position is still available for the repair version of the chem launcher, using the same controls as before.
*** Unequipping the Chem launcher can be done using the skill button, the B (XBOX) / Circle (PS4) button or just using the weapon swap controls.

* Loot & Rewards
** Control Point reward containers are giving the correct number of items for the corresponding Control Point's defense level. Defense level 4 now awards 5 items instead of 3.
** Reduced DZ XP gain from Landmarks in Dark Zones.
** Reduced the quality and power for items dropped in Propaganda Broadcast and one drop of Settlement Blockade activities to be more in line with other open world enemies and activities.
** Updated XP rewards for low-level player playing in World Tiers to scale with player level.
** Supply Drop activity loot container items now scale with every consecutive container you open. This includes lowering the reward of the first crate, but results in similar rewards for the activity as a whole.
** Added 5 Specialized Apparel Cache Keys, awarded with levels 2 – 4.
*** As a thank you for playing, all players that played before April 4 get 5 Specialized Apparel Keys, regardless of their level.
*** Apparel Keys will be coming for those players later next week.
** Faction Key loot drops from Bounties are now guaranteed and are displayed in the loot preview.
** Adjusted bounty credit rewards to use scaling values in World Tiers instead of flat values so if a lower level player is playing with a higher-level player, they will receive an amount of credits more appropriate for their level.
** Changed Skill Unlock rewards to be non-invaded only so that players who co-op with a higher-level friend to play invaded before playing on story won't receive the rewards.
** Added Apparel cache key fragments to the PvP reward caches.
** Adjusted the rewards for the street execution activity to give slightly lower quality/source level gear so the rewards are more in line with the difficulty of the activity.
** Reduced the quality and source level charts used for item drops from the living world activities settlement blockade and Propaganda broadcast to be more in line with other open world drops and the difficulty of the activities.
** Updated Collectibles set XP reward to scale with player level.
** Fixed an issue which caused the Drone Helicopter to not use the correct loot table.

* Crafting 
** Increased Material cap for Standard and Specialized crafting materials to 400 (with all perks).
** Increased Material cap for High-End crafting materials to 50.
** Materials required for crafting weapon mods does not scale as aggressively and has much lower values in endgame.
** Removed a small increase on materials required for crafting weapon mods after upgrading the crafting bench quality.
** Changed the upgrade blueprints for exotic weapons Lullaby and Ruthless to not show up on the vendor until you are level 30.
** Increased the power for the wandering vendor so her inventory has a narrower Gear Score range closer to the top of the World Tier Gear Score Range.
** Set all exotic upgrade blueprints to award you the upgraded weapon on the max gear score of the current tier.
** Removed a tiny increase on materials required for crafting weapon mods after upgrading the crafting bench quality.
** Exotic weapons are now pre-equipped with lore text mods. These mods do not have stats and cannot be equipped elsewhere.

* Talents
** Blind Justice – Modified the Buff that increases damage to your next shot now expires at end of combat and when out of combat
** First Blood - Updated Normalized MMR Talent First Blood Text to correctly explain that the first shot after reloading from empty is the enhanced bullet.

* NPCs
** Decreased damage of all NPC Automatic Shotguns
** Decreased damage of all NPC SMGs
** Hey agent! Staff in the Base of Operation will now shout at the player less frequently than before.

* Main Missions Difficulty
** World Bank – Invaded – Server Room
*** ​Increased the time which the Black Tusks needs to complete the download in the Server Room from 200 seconds to 250 seconds.
*** Reduced the difficulty in the Server Room by setting the Enemy group composition to light, light, medium - Instead of medium, heavy, medium.
** DCD Headquarters
*** Mission level has been changed from 17 to 18.
** Roosevelt Island – Invaded – Quarantine Docks
*** Reduced the difficulty of the first wave of enemies encountered in the last beat of the mission by lowering the amount of Elite and Veteran enemies.
*** Enemies in this part of the mission can only use mounted weapons if there are 2 players or more.
**** This should make it less punishing for solo-players attempting this mission.

* Open World
** Removed death of civilian squad as a failure condition for Control Point takeover.
*** The Officer will now remain in downed state until players complete the takeover or wipe.
** Black Tusk presence has been increased in invaded Open World zones.
** New Ambusher NPCs now roam the streets of DC.

* Social
** Added a “Friends & Clan” option for Call for Backup that only shows calls from friends and clan members.
** Added a “Do Not Disturb” group privacy option that blocks all incoming invites.

* PC
** Added a Field of View slider

* PlayStation 4
** Fixed an issue that could cause players to be stuck in the Specialization tutorial

* Bug Fixes
** Fixed an issue that could cause undesired appearance changes to agents after login in or using the barber.
*** Players that have been permanently affected by an appearance change will unfortunately not be changed back with this fix. We want to apologize for any inconvenienced caused by this and we’re looking into options to allow players to fully customize their agent in the future, but we do not have a timeline for this feature.
** Fixed an issue where the Cluster Seeker Mine could split up more often than intended.
** Fixed an issue where the Defender Drone could have unlimited uptime.
** Fixed an issue which prevented players from finishing the Unite cypher riddle.
*** We're confident this is not bugged anymore.
** Fixed an issue that would cause the Stronghold activation animation to always play when entering the Base of Operations.
** Fixed several instances of Audio logs not playing automatically when picked up.
** Fixed an issue that could cause players being unable to finish the Side Mission Worksite Community.
** Fixed an issue where players were unable to update their crafting bench after reaching a new World Tier.
** Fixed an issue with NPCs sometimes not reacting properly to cooked grenades
** Fixed an issue with Hunters not always appropriately despawning if a player flees a combat space
** Fixed rare issue where NPCs could get stuck when attempting to melee a player in cover
** Fixed issue with the Explosive Seeker Mine occasionally failing to path to its target
** Fixed bug where players could fast travel to non-friendly Control Points
** Fixed bug that made encounters with friendly NPCs requesting resources too rare
** Fixed bug where underground enemies would come to the surface during the day
** Fixed an issue where players were able to obtain gear with a higher gear score than intended
*** Any owned High-End items will be lowered to Gear Score 500.
*** Any owned Superior items will be lowered to Gear Score 490.
*** Clan caches no longer award items higher than the maximum loot Gear Score.
*** Recalibration still allows Gear Score to increase slightly past the maximum loot Gear Score.
** Fixed a bunch of weird stairs and invisible walls.

## Patch from 28. March 2019

* Performance
** Various improvements to reduce frame drops.

* Clans
** Changed the Clan Member UI colour to not conflict with Elite enemies.
** Fixed an issue where players Clan XP contributions could stop registering.

* VOIP
** Changed Push-to-Talk to be the default VOIP setting.

* UI
** Changed Apparel Store purchase confirmations to a button hold action.
** Reduced the UI flickering effect present in the menu.
** Fixed a visual bug that would incorrectly display a grey duplicate backpack reward when opening apparel caches.
** Fixed the menu distortion animation playing twice under certain circumstances.

* Skills
** Fixed abnormal skill behaviour when equipping a skill after having switched to a grenade.
** Fixed instances of skills going on cool down when deploying a shield.
** Fixed an issue where multiple Cluster Seeker Mines could be deployed in short succession.

* Mods
** Fixed an issue where donating a weapon with mods attached would remove the mods from the players inventory.

* Gameplay
** Fixed daily and weekly missions not resetting for offline players.
** Improvements made to Co-op health scaling.
** Fixed an issue where armour/gear scaling would not take World Tiers into account.
** Fixed skill damage incorrectly being scaled twice for low level players joining a high level group.
** Fixed players potentially becoming stuck on the “Meet The Division Coordinator” objective after having played in Co-op.
** Fixed players potentially becoming stuck on the “Meet with Odessa Sawyer” objective.

* Open World
** Fixed an issue that would allow players to upgrade neutral control points.

* Side Missions
** Fixed an issue that would cause low level players to gain unintended amounts of XP by completing side missions with a high level group member.

* PC
** Fixed an issue where some Logitech headphone users experienced missing in-game audio.

* UI
** Fixed “Navigate Categories” and “Mark as Junk” being bound to the same button in some occurrences.

* Xbox One
** Fixed an issue where the HDR setting slider would not move.

## Patch from 22. March 2019

* Fixed an issue where players were not being granted Clan XP
* Fixed an issue where daily and weekly resets would not apply to players who were offline during the time of reset
* Fixed an issue where players could lose guiding missions and access to guiding mission NPCs, making them unable to further progress
* Improved survivability of low level players against enemy NPCs when playing co-op with a player of higher level

## Patch from 16. March 2019

* Bug fixes
** Implemented a fix that should reduce occurrences of client crashes when using skills
** Fixed an issue where players could be stuck in the wrong World Tier if they completed Strongholds without meeting the required Gear Score while in a group
** Fixed an issue where players wouldn't be able to talk to agent Kelso at the Castle Settlement
** Fixed an issue where players could obtain multiple Ivory Keys from the same source
** Fixed an issue where map progress would not be saved when completing Open World activities and Side missions in another players’ instance.
** Fixed an exploit that allowed players to loot Supply Drops multiple times
** Fixed a bug where players could buy blueprints repeatedly at the vendors by logging out and back in
** Fixed an issue where combining the Dialed in talent with the Emet Mask could lead to greatly increased Accuracy and Stability

## Patch from 14. March 2019

* Weapons
** We removed a work in progress weapon that was not properly named and balanced.
** Crossbows now deal explosive damage to multiple objects as intended
** Exotic weapons equipped with silencers will now produce the sound they’re intended to make.
** NPCs engaged in combat will now react to grenades as intended.

* Missions
** The ropes will now deploy properly to allow for extraction at the end of the Beekeeper Jeff side mission.
** Players should no longer be able to unlock Castle prematurely. This issue would block progression in some instances.
** In Jefferson Plaza, the NPC scene can no longer be interrupted during a boss fight.
** Fixed a loot exploit that involved repeatedly killing bosses in some Invaded missions.

* PC specific
** PC players playing together while connected to the same network were meeting a number of issues when interacting with each other. This has now been fixed.
** On PC, FreeSync 2 HDR can now properly be selected in game regardless of your PC’s configuration.

* Other
** Stopped "SHD CPU V.2" and "Cyclone Magazine" skill mods from dropping as loot. Player's who already have the items can use them once they unlock the corresponding specialization perk.
** Players will no longer get stuck on black screens after watching introduction cinematics.
** We have updated XP rewards for all end game activities, so that they will scale to the player’s level, instead of being flat values.
** Several corrupted audio issues have been fixed.

## Title update 1.5

* Title Update 1.5 mainly addresses occurrences of audio missing or cutting out experienced by some during Open Beta.
